db/migrate/20160324174132_create_container_deployments.rb

```diff
@@ -0,0 +1,25 @@
+class CreateContainerDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string     :deployment_type
+      t.string     :deployment_version
+      t.boolean    :containerized, :default => true
+      t.string     :deployment_method
+      t.string     :ssh_private
+      t.string     :ssh_public
```

Comment:
Authentication table doesn't have a  ssh_pub and ssh_private columns, under what columns are they stored?